### ISSUE TITLE
How to change batch size, max number of epochs, learning rate, etc?

### ISSUE THREAD

**Issue by gullalc** [thtrieu/darkflow#463]
I am training yolo on my own dataset with a separate config file. 
I have specified different training parameters in the config file, but training starts with a fixed batch size of 16, learning rate of 1e-5, and maximum epochs of 2000.

Do I have to make changes in the darkflow code to make these changes?
If yes, whats the point of having a config file.

**Comment by txdat** [comment]
you can use darkflow/default.py options in terminal, options in config file not effect

**Comment by billyes** [comment]
Yes same problem here. It seems like to be a bug. 
Just use the command in terminal. Refer to flow --h.

**Comment by gullalc** [comment]
Thanks for the clarification.
